src/deye_events.py

```diff
@@ -77,6 +75,61 @@ def __hash__(self):
         return hash(self.online)
 
 
+class DeyeEventList(list):
+    """
+    An list of Deye Events
+    """
+
+    def __init__(self, events: list[DeyeEvent] | None = None):
+        self.__log = logging.getLogger(DeyeEventList.__name__)
+        super().__init__(events if events else [])
+
+    def __str__(self) -> str:
+        return ", ".join([str(e) for e in self])
+
+    def get_status_event(self) -> DeyeLoggerStatusEvent | None:
+        """Get first status event from event list"""
+        for event in self:
+            if isinstance(event, DeyeLoggerStatusEvent):
+                return event
+        return None
+
+    def is_offline(self) -> bool:
+        """Check for the status event offline
+
+        Returns
+        -------
+        bool
+            True if status event found and is 'offline'.
+        """
+        try:
+            return not self.get_status_event().online
+        except AttributeError:
+            return False
+
+    def compare_observation_events(self, events) -> bool:
+        """
+        Compare observation events of self with other DeyeEventList, ignoring the order of events
+
+        Parameters
+        ----------
+        events : list[DeyeEvent]
+            Other list of events
+        check_status : bool
```

Comment:
Please remove this unused argument

Reply:
Thanks, forgot to remove it as I removed the argument.